addons/account_payment/views/account_portal_templates.xml

```diff
@@ -60,6 +60,11 @@
     </template>
 
     <template id="portal_invoice_page_inherit_payment" name="Payment on My Invoices" inherit_id="account.portal_invoice_page">
+        <xpath expr="//t[@t-if='early_payment']" position="after">
+            <t t-elif="invoice.payment_state in ('paid', 'in_payment')">
+                <span t-esc="invoice.amount_paid" t-options='{"widget": "monetary", "display_currency": invoice.currency_id}' />
+            </t>
+        </xpath>
```

Comment:
Not sure this is working, where is it displayed ?

Reply:
It's a relique from Stefan's PR. I just checked and I don't see why he put it either. Thanks!